pom.xml

```diff
@@ -174,10 +174,23 @@
                 <version>1.23.0</version>
                 <configuration>
                     <java>
+                        <excludes>
+                            <!-- exclude beam patch from formatting -->
+                            <exclude>src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java</exclude>
+                        </excludes>
+                        <licenseHeader>
+                            <!-- end license header with extra trailing newline to separate from package statement -->
+                            <file>${maven.multiModuleProjectDirectory}/checkstyle/java.header</file>
+                        </licenseHeader>
```

Comment:
Does this mean spotless is going to add our header for us automatically?!!!!!

cc @mreid-moz this is worth at least 10 points toward H2 IAP bonus.